api/src/main/java/com/valtech/aem/saas/api/bestbets/BestBetsConsumerService.java

```diff
@@ -0,0 +1,67 @@
+package com.valtech.aem.saas.api.bestbets;
+
+import java.util.List;
+import lombok.NonNull;
+
+/**
+ * Represents the best bets consumer service. Offers the following actions: - add a single best bet - add multiple best
+ * bets - update the best bet - delete a best bet - publish best bets for project - get all best bets
+ */
+public interface BestBetsConsumerService {
+
+  /**
+   * Adds a best bet entry in saas admin.
+   *
+   * @param bestBetPayload object containing details for the best bet.
+   * @throws BestBetsActionFailedException exception thrown when the add action has failed or request execution has
+   *                                       failed
+   */
+  void addBestBet(@NonNull BestBetPayload bestBetPayload);
+
+  /**
+   * Adds a list of best bet entry in saas admin.
+   *
+   * @param bestBetPayloadList a list of objects containing best bet details.
+   * @throws BestBetsActionFailedException exception thrown when the add action has failed
+   * or request execution has failed
+   */
+  void addBestBets(@NonNull List<BestBetPayload> bestBetPayloadList);
+
+  /**
+   * Updates the best bet entry with the specified betId.
+   *
+   * @param betId          id of the best bet that is updated.
+   * @param bestBetPayload best bet details to be updated.
+   * @return the id of the best bet entry that is updated.
+   * @throws BestBetsActionFailedException exception thrown when the update action has failed
+   * or request execution has failed
+   */
+  int updateBestBet(int betId, @NonNull BestBetPayload bestBetPayload);
```

Comment:
general: the return statement is king of useless, it matches always the input. 
- [ ] please rename betID to bestBetd or just id. we shouldn't add new terminology. Also change accordingly javadoc and implementing methods / variable names